tfs/handler.py

```diff
@@ -107,15 +107,16 @@ def read_tfs(tfs_file_path: Union[pathlib.Path, str], index: str = None) -> TfsD
     Parses the TFS table present in tfs_path and returns a custom Pandas DataFrame (TfsDataFrame).
 
     Args:
-        tfs_file_path: pathlib.Path of the input TFS file, can be str but will be cast to pathlib.Path
+        tfs_file_path: PosixPath object to the output TFS file. Can be a string, but will be cast
+        to a PosixPath object.
         index: Name of the column to set as index. If not given looks for INDEX_ID-column
 
     Returns:
         TfsDataFrame object
     """
     tfs_file_path = pathlib.Path(tfs_file_path)
     LOGGER.debug(f"Reading path: {tfs_file_path.absolute()}")
-    headers = OrderedDict()
+    headers = {}
```

Comment:
Why the swap? In the ```__init__``` you swap them the other way around?

Reply:
Was a matter of python 3.6+ or 3.7+